_haikus/favorite.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: favorite
+author: Misaaa
+---
+
+I like reading books
```

Comment:
To create a linebreak in markdown, please add two spaces — `  ` — or a `<br>` tag after each line in this haiku